GUI/SearchTaskWindow.xaml.cs

```diff
@@ -729,7 +729,26 @@ private void NonSpecificUsingNonSpecific(object sender, RoutedEventArgs e)
             }
         }
 
-        private void NonSpecificUpdate(object sender, TextChangedEventArgs e)
+        //this one is used by the GUI
+        private void ProteaseSpecificUpdate(object sender, SelectionChangedEventArgs e)
+        {
+            const int maxLength = 25;
+            if (((Protease)ProteaseComboBox.SelectedItem).Name.Contains("non-specific"))
```

Comment:
should the proteases be enum type? in mzlib?

Reply:
No, they are complex objects that include digestion rules.